org.radare.Cutter.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.radare.Cutter.desktop</id>
```

Comment:
Should `.desktop` be here? Also should appdata provide <launchable>? Trying to figure out CI failure.